src/main/java/frc/trigon/robot/subsystems/coralintake/CoralIntakeConstants.java

```diff
@@ -167,22 +134,22 @@ public class CoralIntakeConstants {
             Color.kRed
     );
 
-    public static final double COLLECTION_LEDS_BLINKING_SPEED = 0.5;
-    private static final double CORAL_COLLECTION_CONFIRMATION_TIME_THRESHOLD_SECONDS = 0.65;
+    public static final double
+            COLLECTION_LEDS_BLINKING_SPEED = 0.5,
+            COLLECTION_RUMBLE_DURATION_SECONDS = 0.7,
+            COLLECTION_RUMBLE_POWER = 1;
+    private static final double
+            CORAL_COLLECTION_DETECTION_DEBOUNCE_TIME_SECONDS = 0.65,
+            EARLY_CORAL_COLLECTION_DETECTION_DEBOUNCE_TIME_SECONDS = 0.06;
     static final BooleanEvent CORAL_COLLECTION_BOOLEAN_EVENT = new BooleanEvent(
             CommandScheduler.getInstance().getActiveButtonLoop(),
             BEAM_BREAK::getBinaryValue
-    ).debounce(CORAL_COLLECTION_CONFIRMATION_TIME_THRESHOLD_SECONDS);
-    private static final double
-            EARLY_COLLECTION_DETECTION_DISTANCE_CENTIMETERS = RobotHardwareStats.isSimulation() ? 100 : 17,
-            EARLY_COLLECTION_DETECTION_TIME_THRESHOLD_SECONDS = 0.06;
+    ).debounce(CORAL_COLLECTION_DETECTION_DEBOUNCE_TIME_SECONDS);
     static final BooleanEvent EARLY_CORAL_COLLECTION_DETECTION_BOOLEAN_EVENT = new BooleanEvent(
             CommandScheduler.getInstance().getActiveButtonLoop(),
-            () -> DISTANCE_SENSOR.getScaledValue() < EARLY_COLLECTION_DETECTION_DISTANCE_CENTIMETERS
-    ).debounce(EARLY_COLLECTION_DETECTION_TIME_THRESHOLD_SECONDS);
-    public static final double
-            COLLECTION_RUMBLE_DURATION_SECONDS = 0.7,
-            COLLECTION_RUMBLE_POWER = 1;
+            () -> DISTANCE_SENSOR.getScaledValue() < (RobotHardwareStats.isSimulation() ? 100 : 17)
```

Comment:
no reason this shouldn't be the same for sim and not sim 
make the maximum distance a constant